src/components/app/App.js

```diff
@@ -1,9 +1,22 @@
-function App() {
+import React, { Component } from 'react';
+import '../../style/global.scss';
+import '../../style/container.scss';
+import '../../style/button.scss';
+import '../../style/variables.scss';
+// import SingleComicsPage from '../../pages/singleComicsPage/SingleComicsPage';
+// import SingleCharacterPage from '../../pages/singleCharacterPage/SingleCharacterPage';
+import MainPage from '../../pages/mainPage/MainPage';
+// import ComicsListPage from '../../pages/comicsListPage/ComicsListPage';
+
+class App extends Component {
+  render() {
     return (
-      <div className="app">
-          
-      </div>
+      <MainPage></MainPage>
```

Comment:
Когда нет ничего между тегами, как у тебя тут, то можно писать сокращенно <MainPage />
```suggestion
      <MainPage />
```